Resort.java

```diff
@@ -224,6 +224,10 @@ private void loadPasses() {
         passes.add(new Pass(1006, "Raj", 4, 5));
         passes.add(new Pass(1007, "Sol", 7, 20));
         passes.add(new Pass(1008, "Tel", 6, 30));
+
+        passes.add(new TouristPass(3,"Killjoy",10,10,"Ukraine"));
```

Comment:
Bruh, that's a tourist name. Also make sure these new passes are formatted correctly and remove the newline on the line above.